pkg/api/v1/defaults.go

```diff
@@ -369,12 +369,15 @@ func SetDefaults_RBDVolumeSource(obj *v1.RBDVolumeSource) {
 }
 
 func SetDefaults_ScaleIOVolumeSource(obj *v1.ScaleIOVolumeSource) {
-	if obj.ProtectionDomain == "" {
-		obj.ProtectionDomain = "default"
+	if obj.StorageMode == "" {
+		obj.StorageMode = "ThinProvisioned"
 	}
-	if obj.StoragePool == "" {
-		obj.StoragePool = "default"
+	if obj.FSType == "" {
+		obj.FSType = "xfs"
 	}
```

Comment:
restore these defaults to both `SetDefaults_ScaleIOVolumeSource` and `SetDefaults_ScaleIOPersistentVolumeSource`?

Reply:
See comment above.   Thanks!